<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>mainWindow</class>
 <widget class="QWidget" name="mainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>436</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget" >
   <property name="geometry" >
    <rect>
     <x>90</x>
     <y>10</y>
     <width>341</width>
     <height>271</height>
    </rect>
   </property>
   <widget class="QWidget" name="tab" >
    <attribute name="title" >
     <string>Tab 1</string>
    </attribute>
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>8</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item row="0" column="0" >
      <widget class="QListWidget" name="listWidget" >
       <item>
        <property name="text" >
         <string>qwepowefklwenmlkwefweklfnwelfknwlfknweklfnwelfknwefwefopjiwelpkwmjvlkwneklwemflkwemdfklwemflwekmfwkled;qweef</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>wefwefweklfl;m,we;lf,;welcv,wel;wekfm;kf;pwecmlwkewl;efmwklegfhweklnqwekldhjl;dmjqwlekdfjqweklf</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>New Item</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>New Item</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>New Item</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>New Item</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>New Item</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>New Item</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>New Item</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>New Item</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>New Item</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>New Item</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>New Item</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>New Item</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>New Item</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>New Item</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>New Item</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>New Item</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>New Item</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>New Item</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>New Item</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>New Item</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>New Item</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>New Item</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>New Item</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>wef[pwoejfweopfjero'wepofjkwepfm,weopfkweopfmkdfokweopf,wepofweopfkweopfkweopfkweopfkwef</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="tab" >
    <attribute name="title" >
     <string>Tab 2</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QPushButton" name="exitButton" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>70</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="updateButton" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>40</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>Update</string>
   </property>
  </widget>
  <widget class="QPushButton" name="searchButton" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>Search</string>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider" >
   <property name="geometry" >
    <rect>
     <x>110</x>
     <y>350</y>
     <width>160</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox" >
   <property name="geometry" >
    <rect>
     <x>250</x>
     <y>310</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox" >
   <property name="geometry" >
    <rect>
     <x>50</x>
     <y>300</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QSlider" name="verticalSlider" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>250</y>
     <width>16</width>
     <height>160</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateTimeEdit" >
   <property name="geometry" >
    <rect>
     <x>110</x>
     <y>390</y>
     <width>194</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources/>
 <connections/>
</ui>
